src/d3/vectorFeature.js

```diff
@@ -48,32 +49,100 @@ geo.d3.vectorFeature = function (arg) {
    * @param {object[]} data The vector data array
    * @param {function} stroke The stroke accessor
    * @param {function} opacity The opacity accessor
+   * @param {function} originStyle The marker style for the vector head
+   * @param {function} endStyle The marker style for the vector tail
    */
   ////////////////////////////////////////////////////////////////////////////
-  function updateMarkers(data, stroke, opacity) {
+  function updateMarkers(data, stroke, opacity, originStyle, endStyle) {
+
+    var markerConfigs = {
+      'arrow': {
+        attrs: {'class': 'geo-vector', 'viewBox': '0 0 10 10', 'refX': '1', 'refY': '5', 'markerHeight': '5', 'markerWidth': '5', 'orient': 'auto'},
```

Comment:
These issues are fixed. I'll add tests to confirm. It was just a minor css issue that caused the d3 selection to miss the already existing markers.